### PR TITLE
webui: Updated bootgrid + added syslog colouring based on priority

### LINKED ISSUE #4270: Syslog coloring rules
Hi
Most of my experience with network monitoring comes from Solarwinds. So I have one suggestion. Is it possible to color syslog messages according to severity level? Something like that:
clipboard_e85f032f7e2b4a7bb6f57cf70733edaee
Now LibreNMS uses small icons, but they are not catch the eye as color and it is easy to miss something important. Of course I can use filter, but it is a different story

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

This updates jquery-bootgrid. I've removed a lot of the surplus files that come with this as it's just unnecessary bloat. If we managed via composer or similar it would be ok but there is no point in us shipping unused files.

The new version of bootgrid was needed, it now provides a map feature to color rows based on the response in the status field.

I've tested alerts, syslog, devices, eventlog tables and they all still work ok.

Produces colouring:

image

Fixes #4270

### CODE REVIEW

File: includes/common.php [hunk 1]
```diff
@@ -33,6 +33,23 @@ function generate_priority_icon($priority)
     return '<img src="images/16/' . $map[$priority] .'.png" title="' . $priority . '">';
 }
 
+function generate_priority_status($priority)
```
Comment (murrant) [inline includes/common.php]: A switch would be better I think...

File: includes/common.php [hunk 2]
```diff
@@ -33,6 +33,23 @@ function generate_priority_icon($priority)
...
+{
+    $map = array(
```
Comment (murrant) [inline includes/common.php]: http://docs.librenms.org/General/Contributing/

**Comment by murrant** [comment]
Probably could dump the jquery.bootgrid-1.3.1.zip too.

**Comment by laf** [comment]
Updated - hopefully the functions have the fall back in place now.

**Comment by murrant** [comment]
@laf Only one commit here ;)

**Comment by laf** [comment]
What do you mean @murrant :whistles:

**Comment by realnakrul** [comment]
Hi. Thank you so much. But Recent syslog on the device page is not colored?

**Comment by laf** [comment]
please create an issue and show the output of validate.

**Pull request merged**